### PR TITLE
Add FORCAM FORCE Bridge connector artifacts

### PR DESCRIPTION
**Pull request by dominikamann**
Initial commit for FORCAM FORCE Bridge Connector

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

File: certified-connectors/FORCAM FORCE Bridge/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,5073 @@
...
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/tickets/ticketId/{ticketId}": {
+      "get": {
+        "description": "Get ticket by Id",
+        "summary": "Get ticket by Id ",
```
Comment (sriyen-msft) [inline certified-connectors/FORCAM FORCE Bridge/apiDefinition.swagger.json:2505]: @dominikamann Please remove extra space at the end of the string

File: certified-connectors/FORCAM FORCE Bridge/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,5073 @@
...
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/operations/dispatched/workplaceId/{workplaceId}": {
+      "get": {
+        "description": "Get operations that are dispatched",
+        "summary": "Get dispatched operations ",
```
Comment (sriyen-msft) [inline certified-connectors/FORCAM FORCE Bridge/apiDefinition.swagger.json:1576]: @dominikamann Please remove extra space at the end of the string

File: certified-connectors/FORCAM FORCE Bridge/apiDefinition.swagger.json (hunk 1 above)
Comment (dominikamann) [reply certified-connectors/FORCAM FORCE Bridge/apiDefinition.swagger.json]: @sriyen-msft Resolved, thank you for the information.

File: certified-connectors/FORCAM FORCE Bridge/apiDefinition.swagger.json (hunk 2 above)
Comment (dominikamann) [reply certified-connectors/FORCAM FORCE Bridge/apiDefinition.swagger.json]: @sriyen-msft Resolved, thank you for the information.

**Pull request merged**